packages/docs/vite.config.mts

```diff
@@ -206,15 +206,20 @@ export default defineConfig(({ command, mode, isSsrBuild }) => {
       // https://github.com/antfu/vite-plugin-pwa
       VitePWA({
         srcDir: 'src',
-        filename: 'service-worker.js',
+        filename: 'sw.js',
         strategies: 'injectManifest',
         includeAssets: ['favicon.ico'],
+        injectRegister: false,
         injectManifest: {
-          globIgnores: ['**/*.html'],
-          additionalManifestEntries: [
-            { url: '/_fallback.html', revision: Date.now().toString(16) },
-          ],
-          dontCacheBustURLsMatching: /assets\/.+[A-Za-z0-9]{8}\.(js|css)$/,
+          minify: false,
+          enableWorkboxModulesLogs: true,
```

Comment:
we're not using  `workbox`

Reply:
VitePWA is built on top of workbox

Reply:
no, you can use it without using any workbox module in the sw and in the client

Reply:
check the 4th task in pending task list

Reply:
we only use here `injectManifest` module in the build to add the injection point (MANIFEST) in the sw.js

Reply:
sorry, the comment seems conflicting

Reply:
I'll try to explain, there are 3 `workbox` packages types:
- runtime
  - client logic for sw registration and interaction: `workbox-window` 
  - sw packages with some recipes to simplify sw logic like `workbox-expiration`, `workbox-precaching`, there are more
- build time (node): `workbox-build` package

We're using here only `workbox-build` since we need the injection point for manual sw precaching logic. If you disable the `injectionPoint` in the `injectManifest` pwa option (`injectionPoint: undefined`), the Vite PWA plugin will not use any `workbox` module.

Reply:
yeah, get that, i mean the initial comment about `enableWorkboxModulesLogs: true` -> "we're not using `workbox`" on the surface seemed to contradict each other. thanks for the clarification.

Reply:
Both, `workbox-build::generateSW` and `workbox-build::injectManifest` loaded with dynamic import and when required:
- `injectManifest` strategy in pwa options: the custom Vite build will omit the `workbox-build::injectManifest` call if no injection point https://github.com/vite-pwa/vite-plugin-pwa/blob/main/src/vite-build.ts#L61-L81
- `generateSW` strategy in pwa options: https://github.com/vite-pwa/vite-plugin-pwa/blob/main/src/modules.ts#L80-L92